server/models/index.js

```diff
@@ -7,7 +7,7 @@ const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
 const config = configPath[env];
 
-const db = {};
+export const db = {};
```

Comment:
For the sake of consistency, changed this line to use ES6 export syntax as opposed to module.exports syntax